### ISSUE TITLE
[bug] ANSI escape sequences 

### ISSUE THREAD

**Issue by Lutra-Fs** [gjwgit/rattle#359]
**Description:**

The ANSI escape sequence `[38;5;246m[3m` is not being processed correctly, resulting in incorrect text formatting.

**Expected behavior:**

The sequence should change the text color to a medium gray (color code 246) and enable italic text.

**Actual behavior:**

The sequence is not being interpreted correctly, and the text is not being displayed with the expected formatting. 

**Steps to reproduce:**

1. Load a data set
3. Observe the resulting text formatting between Dataset Glimpse page and console tab

**Environment:**

* Version: 6.2.51 - current
* Operating System: Windows 11

**Additional context:**

* The sequence is being output by `dplyr::glimpse(ds)` as part of its output.

**Example output:**

Internal Forum Post: transferred from https://wattlecourses.anu.edu.au/mod/forum/discuss.php?d=962608#p2487941, see it for more details.

**Comment by Lutra-Fs** [comment]
https://github.com/gjwgit/rattleng/commit/232630bd8a4f7cfc62dd824c8b2e50ceaf27bbc5

 I noticed that the ANSI escape sequence for colourisation has been disabled. That's an interesting change! I wonder if the sequences have now been processed correctly. Many thanks.

**Comment by gjwgit** [comment]
Yes, I turned off the ANSI escape sequences generated from R - they seemed to keep causing issues and seems better to eliminate them from the string processing as they play no role in that. Hopefully this fixes the Windows stdout processing and eliminates a lot of the cleaning in utils/clean_string.dart. I also did not incorporate your more general ANSI processing, with this as possibly a simpler solution. 

We can test removing the specific ANSI cleaning from clean_string.dart. If you have a chance on Windows that would be useful - which lines can we now remove from clean_string.dart.

**Comment by Lutra-Fs** [comment]
> Yes, I turned off the ANSI escape sequences generated from R - they seemed to keep causing issues and seems better to eliminate them from the string processing as they play no role in that. Hopefully this fixes the Windows stdout processing and eliminates a lot of the cleaning in utils/clean_string.dart. I also did not incorporate your more general ANSI processing, with this as possibly a simpler solution.
> 
> We can test removing the specific ANSI cleaning from clean_string.dart. If you have a chance on Windows that would be useful - which lines can we now remove from clean_string.dart.

Actually, it is difficult to test because we cannot predict which sequence will appear/

**Comment by Lutra-Fs** [comment]
![image](https://github.com/user-attachments/assets/e582343a-7722-4f91-a1af-8ae911e6fa82)

Can see this happens to this page while doing NN integration test

**Comment by gjwgit** [comment]
Hi @Lutra-Fs, Where do you see this happening again?

**Comment by Lutra-Fs** [comment]
> Hi @Lutra-Fs, Where do you see this happening again?

While running the integration test, I observed the behaviour.

**Comment by gjwgit** [comment]
Can you be more specific? Thanks.

**Comment by Lutra-Fs** [comment]
Cannot manage to get the nn build to work again.

Currently, there is an error in my console:

> form   <- formula(target %s+% " ~ .")
Error in str2lang(s) : argument must be a character string

with the same setting stated in the pic, and so I can only see:
object 'model_nn' not found now...

BTW, I think the command that will reproduce the error will be `flutter test .\integration_test\demo_model_neural_nnet_numeric.dart` under Windows.
